app/views/courses/course.json.jbuilder

```diff
@@ -17,7 +17,7 @@ json.course do
   json.edit_count number_to_human @course.revisions.count
   json.student_count @course.user_count
   json.trained_count @course.students_without_nonstudents.trained.count
-  json.character_count number_to_human @course.character_sum
+  json.word_count number_to_human word_count_from_character_sum @course.character_sum
```

Comment:
Here also. If you want to avoid a double set of parens, it could be `number_to_human word_count_from_character_sum(@course.character_sum)`